.github/workflows/ci.yml

```diff
@@ -49,5 +49,6 @@ jobs:
           name: codecov-umbrella
 
       - name: tmate debug
-        if: ${{ failure() }}
+        continue-on-error: true
+        if: ${{ contains(env.COMMIT_MESSAGE, '[gha-debug]') }}
```

Comment:
The `COMMIT_MESSAGE` isn't populated by default. That's done in a custom way in the OpenMC CI. Let's remove this.